rssfeed/templates/rssfeed/inclusion_tags/entry_list.html

```diff
@@ -0,0 +1,25 @@
+{% load i18n %}
+{% if entry_list %}
+    {% for entry in entry_list %}
+        <div class="feed_entry">
+            <img src="{{ entry.image }}" height="200" width="300">
```

Comment:
You can't assume those sizes will work. You will need to add photologue as a dependency to your app. It will do proper scaling.